### ISSUE TITLE
Weird flickering of items and menus.

### ISSUE THREAD

**Issue by magpie514** [hluk/CopyQ#2766]
**Describe the bug**
Since last release, 9.0.0, items flicker uncomfortably on the main interface, and right-clicking items causes the right-click menu to also flicker, making it unusable (even if you manage to hit an option it won't register). Windows such as the Commands window are fine, though.

**To Reproduce**
In the main menu, selecting a different entry will start flickering (the very first item does not seem to flicker until changed). The right-click menu flickers immediately.

**Expected behavior**
No flickering.

**Screenshots**

**Version, OS and Environment**
- Application Version (for example 7.1.0): CopyQ 9.0.0
- OS (for example Windows 11, Ubuntu 22.04, macOS Sonoma 14.4): Ubuntu 24.04 x86-64
- Desktop environment, window manager (if applicable): KDE Plasma 5 (X11)

**Additional context**
N/A (no visible errors or warnings)
- The flickering item is either the first one, or the selected one. The other items display fine.
- The menu bar's "Item" menu also flickers. The other menus are unaffected and work fine as well.
- If the toolbar is enabled it'll also flicker, and constantly resize the layout.

**Comment by magpie514** [comment]
Turns out the cause is apparently [this command](https://github.com/hluk/copyq-commands/blob/master/Display/toggle-show-as-plain-text.ini), Toggle Show As Plain Text, once I turned it off things started working fine again.

I find this a bit strange since it's in theory an "activated" action, it's (as far as I can tell) not being run on a timer or anything, so it's odd it's forcing an update of the current item a few times per second. It's the only "Display" action I have installed and it never caused issues until 9.0.0.

**My issue is fixed**, but there might be some deeper underlying issue. I'll leave it to your discretion to close or rename this issue as you see fit.
